server/api/scheduler.go

```diff
@@ -160,6 +160,11 @@ func (h *schedulerHandler) CreateScheduler(w http.ResponseWriter, r *http.Reques
 			h.r.JSON(w, http.StatusInternalServerError, err.Error())
 			return
 		}
+	case schedulers.TransferWitnessLeaderName:
```

Comment:
Does it need to handle the balance-witness-scheduler type?

Reply:
Yes, but already handled.